.github/workflows/ci-integration-test-yarn-berry.yml

```diff
@@ -0,0 +1,32 @@
+name: CI - integration test yarn berry
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    name: Nx Cloud - Main Job
+    uses: ./.github/workflows/nx-cloud-main.yml
+    with:
+      working-directory: ./integration-tests/yarn-berry
+      parallel-commands: |
+        npx nx workspace-lint
```

Comment:
I would prefer having here `yarn nx...` over `npx nx...` but it's just nit-picking.